core/templates/pages/admin-page/admin-page.module.ts

```diff
@@ -52,11 +52,13 @@ import { platformFeatureInitFactory, PlatformFeatureService } from
   ],
   declarations: [
     OppiaAngularRootComponent,
```

Comment:
For reviewers: Ideally, we would like to keep this array sorted but since the OppiaAngularRootComponent is temporary we are keeping it at the top. Everything after the line should be sorted. I will add a comment about this too in the module too.

Reply:
Thanks for the explanation @srijanreddy98!